pages/[slug].tsx

```diff
@@ -46,6 +46,7 @@ export const getServerSideProps: GetServerSideProps<BasePageProps> = async (cont
             ...serverSideProps.newsroomContextProps,
             currentStory: story,
         },
+        isTrackingEnabled: isTrackingEnabled(context),
```

Comment:
I wanted to return this property as part of `serverSideProps` but it's defined in Bea and not the theme kit so it didn't feel right.